### PR TITLE
Add miss dependency specs

### PR DESCRIPTION
**Pull request by wushiqinlou**
### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### Complete Description of Additions/Changes:

* Add back the specs removed by mistake in https://github.com/RedHatInsights/insights-core/pull/2663/files
* Remove "lvs_headings, pvs_headings, vgs_headings"

### CODE REVIEW

**Review by xiangce — CHANGES_REQUESTED** [review]
@wushiqinlou - generally, before removing existing parsers, we'd mark them as `deprecated` first. And we'd better split the adding specs and removing specs/parsers into separate PRs.

**Comment by wushiqinlou** [comment]
@xiangce - `deprecated` the existing parsers in https://github.com/RedHatInsights/insights-core/pull/4025, please confirm

File: insights/specs/datasources/dumpdev.py
Comment (xiangce) [inline insights/specs/datasources/dumpdev.py:1]: Sorry for replying late, as we have an exiting python module named `mount` in the `insights/specs/datasources` dir, would you mind moving this new `dumpdev_list` to the existing one? 
Or, it's required to add one more doc entry to the `docs/custom_datasources_index.rst` file.

**Comment by wushiqinlou** [comment]
@xiangce - please check

**Review by xiangce — APPROVED** [review]

**Pull request merged**